### PR TITLE
Improve Domain Model and Data documentation

### PR DESCRIPTION
**Pull request by MarkvanMents**

### CODE REVIEW

File: content/en/docs/refguide/modeling/domain-model/_index.md [hunk 1]
```diff
@@ -8,59 +9,89 @@ description: "Introduces the domain model in Studio Pro."
...
 
-Below is a domain model that defines customers and orders. The names of the entities are `Customer` and `Order`. The line between them is an association. `Order_Customer`. One customer can have many orders, but each order is for one customer. Within the boxes representing the entities you can see the [attributes](/refguide/attributes/) of the entities together with the [type](/refguide/attributes/#type) of data they hold. There is also a [non-persistable](/refguide/persistability/) entity, `ProductQueryResults`, which is used to record product information which is retrieved from a separate product system.
+A domain model consists of [entities](/refguide/entities/) with their relationships to other entities represented by [associations](/refguide/associations/). Entities can be one of three types:
 
-{{< figure src="/attachments/refguide/modeling/domain-model/annotated-domain-model.png" alt="Domain Model annotated with structure" class="no-border" >}}
+* Persistable (blue) – entities which hold app information which needs to be held permanently or shared with other end-users of the app
```
Comment (arjan-h) [inline content/en/docs/refguide/modeling/domain-model/_index.md]: @MarkvanMents Maybe add a link to the section below about Persistable entities

File: content/en/docs/refguide/modeling/domain-model/_index.md [hunk 2]
```diff
@@ -8,59 +9,89 @@ description: "Introduces the domain model in Studio Pro."
...
 | Many | Indicates that many of these entities relate to the quantity of the entity at the other end of the association |
-| Association Owner | An end of an association without an arrow indicates that this entity [owns](/refguide/associations/#ownership) the association (it is also possible for both entities to own the association) |
-| Attribute Name | How this attribute will be referred to in the database |
-| Attribute Type | The [type](/refguide/attributes/#type) of data stored in this attribute |
+| Association Owner | An end of an association without an arrow indicates that this entity owns the association (it is also possible for both entities to own the association, see [ownership](/refguide/associations/#ownership) in *Associations* for more information) |
+| Attribute Name | How this [attribute](/refguide/attributes/) will be referred to in the database, *"Number"* |
+| Attribute Type | The [type](/refguide/attributes/#type) (*Autonumber*) of data stored in this attribute |
 | Non-persistable Entity | This is an entity which is not stored in a database but only stored temporarily within the app |
```
Comment (arjan-h) [inline content/en/docs/refguide/modeling/domain-model/_index.md:46]: @MarkvanMents For completeness and consistency I would expect persistable entities here too (and marked in the image probably)

File: content/en/docs/refguide/modeling/domain-model/_index.md (hunk 1 above)
Comment (MarkvanMents) [reply content/en/docs/refguide/modeling/domain-model/_index.md]: Added in d537bbf9c50965d8da04044942669f3e775511dc

File: content/en/docs/refguide/modeling/domain-model/_index.md (hunk 2 above)
Comment (MarkvanMents) [reply content/en/docs/refguide/modeling/domain-model/_index.md:46]: Added in d537bbf9c50965d8da04044942669f3e775511dc

**Pull request merged**